scripts/codegen-removal/remove_codegen.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -eo pipefail
+CURRENT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"
+
+java -jar "$CURRENT_DIR/codegen-removal.jar" "$@"
```

Comment:
Please, check that java exists. If it isn't installed, then write help, that user should install java8